server/routes/imageUpload.js

```diff
@@ -0,0 +1,28 @@
+const express = require("express");
+const { validate } = require("../middlewares/validate");
+const router = express.Router();
+const upload = require("../services/imageUpload");
+const uploadFile = upload.single("image");
+
+const uploadImage = (req, res) => {
+  uploadFile(req, res, (err) => {
+    if (err) {
+      console.log(err);
```

Comment:
Remember to delete the console.log (if you used it for checking and is not intentional). If it's intentional, it may be better to use the [`console.error()`](https://developer.mozilla.org/en-US/docs/Web/API/Console/error) method instead, since it provides stack-tracing and a nice red color 😈

Reply:
Oh yup, I probably should remove this one as it was for testing when setting up the upload module. Thanks!